applications/app/views/crosswordSearch.scala.html

```diff
@@ -9,7 +9,7 @@
             <header class="content__head tonal__head tonal__head--tone-news">
                 <div class="gs-container">
                     <div class="content__main-column">
-                        <div class="content__labels">
+                        <div class="content__labels content__labels--not-immersive">
```

Comment:
`content__labels` is the common stuff, `content__labels--not-immersive` is the stuff that's not immersive. We have done it like this to avoid CSS overrides, which reduces complexity.